web/src/Sidebar/index.js

```diff
@@ -0,0 +1,104 @@
+
+const options = { year: 'numeric', month: 'long', day: 'numeric' };
+
+class Sidebar extends Component {
+    state = {
+        activeDateKey: null
+    };
+
+    selectDate = (activeDateKey) => () => {
+        this.setState({activeDateKey})
+    };
+
+    changeGraphDetail = (mode, relative) => () => {
+        const {setOptions} = this.context;
+        if (!relative && mode === MODE_LOG_LOG) {
+            mode = MODE_LINEAR;
+        }
+        setOptions({mode, relative});
+    };
+
+    render() {
+        return (
+            <AvailablePredictionsContext.Consumer>
+                {context =>
+                    <GraphDetailContext.Consumer>
+                        {graphDetailContext => {
+                            const dates = [];
+                            const {relative, mode} = graphDetailContext.options;
+
+                            for (let key in context.data) {
+                                const item = context.data[key];
+                                const date = new Date(Date.parse(item.label));
+                                const className = `tab${this.state.activeDateKey === key ? ' active' : ''}`;
+
+                                dates.push(<button key={key} className={className} onClick={this.selectDate(key)}>{date.toLocaleDateString(undefined, options)}</button>);
+                            }
+
+                            return (
+                                <>
+                                    <h1>COVID-19 predictions of Boďová and Kollár</h1>
+                                    <p>Mathematicians Katarína Boďová and Richard Kollár predicted in March and April 2020
+                                    the growth of active cases during COVID-19 pandemic. Their model suggests polynomial
+                                    growth with exponential decay given by:</p>
+                                    <ul>
+                                        <li><em>N</em>(<em>t</em>) = (<em>A</em>/<em>T</em><sub><em>G</em></sub>) ⋅
+                                        (<em>t</em>/<em>T</em><sub><em>G</em></sub>)<sup>α</sup> /
+                                        e<sup><em>t</em>/<em>T</em><sub><em>G</em></sub></sup></li>
+                                    </ul>
+                                    <p>Where:</p>
+                                    <ul><li><em>t</em> is time in days counted from a country-specific "day one"</li><li><em>N(t)</em> the number of active cases (cumulative positively tested minus recovered and deceased)</li><li><em>A</em>, <em>T<sub>G</sub></em> and <em>α</em> are country-specific parameters</li></ul>
+                                    <p>They made two predictions, on March 30 (for 7 countries) and on April 12 (for 23
+                                    countries), each based on data available until the day before. The first prediction
+                                    assumed a common growth parameter <em>α</em> = 6.23.</p>
```

Comment:
Tu by si vedel renderovať markdown? A vedel by si ho zdielať s našími Flask/Dash aplikáciami?

Ako prvý krok ale asi treba dať web spolu do jedného adresára, viď môj ďalší komentár.

Reply:
Viem. Dal by som to vsak ako separatnu issue.